Src/BootCamp.Chapter/Equipment.cs

```diff
@@ -7,67 +7,54 @@ namespace BootCamp.Chapter
     /// </summary>
     public class Equipment
     {
-        private Weapon _weapon;
-        public void SetWeapon(Weapon weapon)
-        {
-
-        }
+        public Weapon Weapon { get; set; }
 
-        private Headpiece _head;
-        public void SetHead(Headpiece head)
-        {
+        public Headpiece Head { get; set; }
 
-        }
+        public Chestpiece Chest { get; set;  }
+        
+        public Shoulderpiece LeftShoulder { get; set; }
 
-        private Chestpiece _chest;
-        public void SetChest(Chestpiece chestpiece)
-        {
+        public Shoulderpiece RightShoulder { get; set; }
 
-        }
+        public Legspiece Legs { get; set; }
 
-        private Shoulderpiece _leftShoulder;
-        public void SetLeftShoulder(Shoulderpiece should)
-        {
+        public Armpiece LeftArm { get; set; }
+        
+        public Armpiece RightArm { get; set; }
 
-        }
-
-        private Shoulderpiece _rightShoulder;
-        public void SetRightShoulder(Shoulderpiece shoulder)
-        {
-
-        }
-
-        private Legspiece _legs;
-        public void SetLeg(Legspiece legs)
-        {
-
-        }
-
-        private Armpiece _leftArm;
-        public void SetLeftArmp(Armpiece arm)
-        {
-
-        }
-
-        private Armpiece _rightArm;
-        public void SetRightArm(Armpiece arm)
-        {
-
-        }
-
-        private Gloves _gloves;
-        public void SetGloves(Gloves gloves)
-        {
-
-        }
+        public Gloves Gloves { get; set; }
```

Comment:
All your get;set; before were just SetX. Please amend getters, as they are not needed here.